LeezeIt/LeezeIt/ViewController.swift

```diff
@@ -9,11 +9,16 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    
+//    @IBAction func goToPhoneScreen(_ sender: Any) {
+//
```

Comment:
Your app is crashing because of this, why did you comment it out?

Reply:
resolved